pages/common/killall.md

```diff
@@ -10,7 +10,7 @@
 
 - List available signal names (to be used without the 'SIG' prefix):
 
-`killall --list`
+`killall -list`
```

Comment:
```suggestion
`killall --list`
```
Welcome to tldr, thanks for your contribution. This isn't a typo here long arguments require 2 `-`. We prefer longer arguments in tldr. 1 `-` will be applicable here for `-l` which is a shorter flag.

Reply:
Thanks for your welcome. In my MBP M1 laptop, `killall --list` is not applicable, while `killall -list` is.

Reply:
<img width="632" alt="image" src="https://user-images.githubusercontent.com/43040147/203210907-f3042bf7-b77d-4f1d-b8e3-492c0b0bc56e.png">

Reply:
> Thanks for your welcome. In my MBP M1 laptop, `killall --list` is not applicable, while `killall -list` is.

Oh, I will check my laptop and get back to you.

Reply:
Update: Both flags seem to work for me. 

Also, their man page has double arguments so not sure.

![image](https://user-images.githubusercontent.com/26346867/203213271-656f59e0-33eb-44aa-a182-dbc55587b004.png)

Reply:
Oh, in my laptop, the man page is:
```
KILLALL(1)                                                               General Commands Manual                                                              KILLALL(1)

NAME
     killall – kill processes by name

SYNOPSIS
     killall [-delmsvqz] [-help] [-I] [-u user] [-t tty] [-c procname] [-SIGNAL] [procname ...]
...
```
while Linux man page for killall is:
```
Name

killall - kill processes by name
Synopsis

killall [-Z,--context pattern] [-e,--exact] [-g,--process-group] [-i,--interactive] [-q,--quiet] [-r,--regexp] [-s,--signal signal] [-u,--user user] [-v,--verbose] [-w,--wait] [-I,--ignore-case] [-V,--version] [--] name ...
killall -l
killall -V,--version 
...
```

The implementations are different.
In this case, I have no strong opinion on any decision of this PR.